src/openrct2/windows/install_track.c

```diff
@@ -246,14 +252,19 @@ static void window_install_track_paint(rct_window *w, rct_drawpixelinfo *dpi)
         }
     }
 
-    // Track design name
-    gfx_draw_string_centred_clipped(dpi, STR_TRACK_PREVIEW_NAME_FORMAT, &_trackName, COLOUR_BLACK, x, y, 368);
-
     // Information
     x = w->x + widget->left + 1;
-    y = w->y + widget->bottom + 2;
+    y = w->y + widget->bottom + 4;
     // 0x006D3CF1 -- 0x006d3d71 missing
 
+    // Track design name & type
+    gfx_draw_string_left(dpi, STR_TRACK_DESIGN_NAME, &_trackName, COLOUR_BLACK, x-1, y);
+    y += 10;
+
+    rct_string_id friendlyTrackName = RideNaming[td6->type].name;
```

Comment:
`get_friendly_track_type_name()`. This will properly return stuff like 'Hyper-Twister'. RideNaming contains the technical track names as used in the Scenario Editor.